.changeset/shaggy-chefs-reply.md

```diff
@@ -0,0 +1,6 @@
+---
+"@pnpm/resolver-base": minor
+"@pnpm/npm-resolver": minor
```

Comment:
```suggestion
"@pnpm/npm-resolver": major
```

Reply:
Ah, I meant to ask you whether returning a new enum value should be a break here.